### PR TITLE
Bugfixes/spell mistakes

### PR DESCRIPTION
**Pull request by gnomeby**

### CODE REVIEW

File: Core/ROMv1.py [hunk 1]
```diff
@@ -102,7 +102,7 @@
 # 2 instruction reset:
 #       ld 0
 #       st [sample]
-# The difference is unhearable. This is fine when the reset/address
+# The difference is not perceptible. This is fine when the reset/address
```
Comment (Cwiiis) [inline Core/ROMv1.py]: 'not audible' would've been a better correction here I think.

File: Core/ROMv1.py [hunk 2]
```diff
@@ -1117,7 +1117,7 @@ def runVcpu(n, ref=None):
 # Enter the timing-aware application interpreter (aka virtual CPU, vCPU)
 #
 # This routine will execute as many as possible instructions in the
-# alotted time. When time runs out, it synchronizes such that the total
+# allowed time. When time runs out, it synchronizes such that the total
```
Comment (Cwiiis) [inline Core/ROMv1.py]: 'allotted' would be the right correction here.

File: Core/ROMv2.py [hunk 3]
```diff
@@ -164,7 +164,7 @@
 # 2 instruction reset:
 #       ld 0
 #       st [sample]
-# The difference is unhearable. This is fine when the reset/address
+# The difference is not perceptible. This is fine when the reset/address
```
Comment (Cwiiis) [inline Core/ROMv2.py]: Same here, 'not audible'.

File: Core/ROMv2.py [hunk 4]
```diff
@@ -1152,7 +1152,7 @@ def runVcpu(n, ref, returnTo=None):
 # Enter the timing-aware application interpreter (aka virtual CPU, vCPU)
 #
 # This routine will execute as many as possible instructions in the
-# alotted time. When time runs out, it synchronizes such that the total
+# allowed time. When time runs out, it synchronizes such that the total
```
Comment (Cwiiis) [inline Core/ROMv2.py]: Same, 'allotted' here.

File: Core/ROMv3.py [hunk 5]
```diff
@@ -150,7 +150,7 @@
 # 2 instruction reset:
 #       ld 0
 #       st [sample]
-# The difference is unhearable. This is fine when the reset/address
+# The difference is not perceptible. This is fine when the reset/address
```
Comment (Cwiiis) [inline Core/ROMv3.py]: And here.

File: Core/ROMv3.py [hunk 6]
```diff
@@ -1142,7 +1142,7 @@ def runVcpu(n, ref, returnTo=None):
 # Enter the timing-aware application interpreter (aka virtual CPU, vCPU)
 #
 # This routine will execute as many as possible instructions in the
-# alotted time. When time runs out, it synchronizes such that the total
+# allowed time. When time runs out, it synchronizes such that the total
```
Comment (Cwiiis) [inline Core/ROMv3.py]: Ditto.

File: Core/ROMv3y.py [hunk 7]
```diff
@@ -154,7 +154,7 @@
 # 2 instruction reset:
 #       ld 0
 #       st [sample]
-# The difference is unhearable. This is fine when the reset/address
+# The difference is not perceptible. This is fine when the reset/address
```
Comment (Cwiiis) [inline Core/ROMv3y.py]: Ditto.

File: Core/ROMv3y.py [hunk 8]
```diff
@@ -1163,7 +1163,7 @@ def runVcpu(n, ref, returnTo=None):
 # Enter the timing-aware application interpreter (aka virtual CPU, vCPU)
 #
 # This routine will execute as many as possible instructions in the
-# alotted time. When time runs out, it synchronizes such that the total
+# allowed time. When time runs out, it synchronizes such that the total
```
Comment (Cwiiis) [inline Core/ROMv3y.py]: Ditto.

File: Core/ROMv4.py [hunk 9]
```diff
@@ -160,7 +160,7 @@
 # 2 instruction reset:
 #       ld 0
 #       st [sample]
-# The difference is unhearable. This is fine when the reset/address
+# The difference is not perceptible. This is fine when the reset/address
```
Comment (Cwiiis) [inline Core/ROMv4.py]: Ditto.

File: Core/dev.py [hunk 10]
```diff
@@ -1256,7 +1256,7 @@ def runVcpu(n, ref=None, returnTo=None):
 # Enter the timing-aware application interpreter (aka virtual CPU, vCPU)
 #
 # This routine will execute as many as possible instructions in the
-# alotted time. When time runs out, it synchronizes such that the total
+# allowed time. When time runs out, it synchronizes such that the total
```
Comment (Cwiiis) [inline Core/dev.py]: Ditto.

**Comment by gnomeby** [comment]
Here you are.

**Comment by kervinck** [comment]
Thanks a million

**Comment by gnomeby** [comment]
You are welcome.

**Pull request merged**